quarkus/pom.xml

```diff
@@ -104,6 +104,10 @@
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-container-image-docker</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-kubernetes-service-binding</artifactId>
+    </dependency>
```

Comment:
Nice. Is there somewhere where [quarkus.kubernetes-service-binding.root or QUARKUS_KUBERNETES_SERVICE_BINDING_ROOT](https://quarkus.io/guides/all-config#quarkus-kubernetes-service-binding_quarkus.kubernetes-service-binding.root) is configured?

Reply:
it assumes default that work

Reply:
Okay, that's great. I was missing the `SBO` part in the title. We do not care about having local development for this right?

Reply:
there is no way to use sbo for local dev I think. The only way would be to manually create folder structure.
Do you see value in having placeholder files committed to the repository?